mnemosyne/core/beam.py

```
@@ -1790,11 +1832,25 @@ def _lexical_relevance(query_tokens: List[str], content: str, query_lower: str =
     if not content_tokens and not query_cjk:
         return 0.0
 
-    exact = sum(1 for token in query_tokens if token in content_tokens)
+    exact = 0
     partial = 0.0
-    for token in query_tokens:
+    for token, components in zip(query_tokens, component_groups):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)